app/classes/query/modules/lookup_objects.rb

```diff
@@ -41,10 +41,9 @@ def lookup_lists_for_projects_by_name(vals)
         project_ids = lookup_projects_by_name(vals)
         return [] if project_ids.empty?
 
-        SpeciesList.connection.select_values(%(
-          SELECT DISTINCT species_list_id FROM project_species_lists
-          WHERE project_id IN (#{project_ids.join(",")})
-        ))
+        SpeciesList.joins(:project_species_lists).
+          where(project_species_lists: { project_id: project_ids }).distinct.
+          pluck(:id)
```

Comment:
This looks good. But I'd suggest generally switching from `pluck(:id)` to `.map(&:id)` following @pellaea's benchmarking. 

I'm amazed you're finding things to convert in the Query class, i was scared to touch it!

Reply:
Thanks for the reminder about pluck vs map.
I will make that change.

Reply:
I actually benchmarked this with a single run of each alternative:
```ruby
[#<Benchmark::Tms:0x00007f92d86b6f30
  @cstime=0.0,
  @cutime=0.0,
  @label="pluck:",
  @real=0.0013135079989297083,
  @stime=3.399999999997849e-05,
  @total=0.0008969999999989264,
  @utime=0.0008629999999989479>,
 #<Benchmark::Tms:0x000055c1ef274ca8
  @cstime=0.0,
  @cutime=0.0,
  @label="map:",
  @real=0.0012400790001265705,
  @stime=2.999999999997449e-05,
  @total=0.0007929999999991555,
  @utime=0.000762999999999181>] 
3.0.4 :028 > project_ids
 => 50..100 
```